maven-plugin/src/main/maven/plugin.xml

```diff
@@ -94,7 +94,8 @@
         <extraGenerators implementation="java.util.List" default-value=""/>
         <protoPaths default-value="${project.basedir}/src/main/proto,${project.basedir}/src/main/protobuf">${akka-grpc.protoPaths}</protoPaths>
         <outputDirectory default-value="${project.build.directory}/generated-sources">${akka-grpc.outputDirectory}</outputDirectory>
-        <protocVersion implementation="java.lang.String" default-value="-v3.11.4">${akka-grpc.protoc-version}</protocVersion>
+        <!-- Keep aligned with Akka gRPC proto-java dependency -->
+        <protocVersion implementation="java.lang.String" default-value="-v3.18.1">${akka-grpc.protoc-version}</protocVersion>
```

Comment:
Couldn't quite figure out if this is picked up from elsewhere somehow or the default is actually used